go/teams/rename.go

```diff
@@ -135,6 +138,15 @@ func generateRenameSubteamSigForParentChain(m libkb.MetaContext, me libkb.UserFo
 		Entropy: entropy,
 	}
 
+	ratchet, err := parentTeam.makeHiddenRatchet(m)
```

Comment:
why do we need to update parent hidden ratchets for rename subteam?

Reply:
Whenever we post anything to a sigchain, the server requires that the "ratchets" are up-to-date. So in my test, I rotate the parent chain, then try to rename a child, and the server is checking that we know about the rotation (via the ratchet) before it accepts the new rename link in the parent chain.